### PR TITLE
Clarify the relationship between network digital twin and the network management

### PR DESCRIPTION
**Pull request by billwuqin**
See Network digital twin as a tool

### CODE REVIEW

File: draft-irtf-nmrg-network-digital-twin-arch.md [hunk 1]
```diff
@@ -257,17 +257,26 @@ in the digital world.  It has the characteristics of virtual-reality
...
-driven approach to build closed-loop network automation.  Therefore,
-a network digital twin platform is more than an emulation platform or
-network simulator.
+acknowledged in academic publications and adopted in Industry 4.0.
+See more in {{concept}}.
+
+A digital twin for networks can be built by applying Digital Twin
+technologies to networks and creating a virtual image of real
+network facilities (called herein, emulation). Basically, the digital
+twin for networks is an expansion platform of network simulation and
```
Comment (boucadair) [inline draft-irtf-nmrg-network-digital-twin-arch.md:266]:
```suggestion
twin for networks is an expansion platform of network emulation and
```

File: draft-irtf-nmrg-network-digital-twin-arch.md [hunk 2]
```diff
@@ -257,17 +257,26 @@ in the digital world.  It has the characteristics of virtual-reality
...
+can be seen as a powerful tool for scenario planning, impact analysis,
```
Comment (boucadair) [inline draft-irtf-nmrg-network-digital-twin-arch.md:267]:
```suggestion
can be seen as a tool for scenario planning, impact analysis,
```

File: draft-irtf-nmrg-network-digital-twin-arch.md [hunk 3]
```diff
@@ -257,17 +257,26 @@ in the digital world.  It has the characteristics of virtual-reality
...
+and change management. The main difference compared to traditional
```
Comment (boucadair) [inline draft-irtf-nmrg-network-digital-twin-arch.md:268]:
```suggestion
and change management. The main difference compared to conventional
```

File: draft-irtf-nmrg-network-digital-twin-arch.md [hunk 4]
```diff
@@ -257,17 +257,26 @@ in the digital world.  It has the characteristics of virtual-reality
...
+network simulation is the interactive virtual-real mapping and data
+driven approach to build closed-loop network automation.  By
+integrating network digital twin into the network management, it
+allows network maintenance engineers to test and model optimisation
```
Comment (boucadair) [inline draft-irtf-nmrg-network-digital-twin-arch.md:272]:
```suggestion
allows network maintenance engineers to assess, model, and tweak optimization
```

File: draft-irtf-nmrg-network-digital-twin-arch.md [hunk 5]
```diff
@@ -257,17 +257,26 @@ in the digital world.  It has the characteristics of virtual-reality
...
+strategies in a risk-free environment, ensuring that only the most
+effective changes are implemented in the real network. Digital twin
```
Comment (boucadair) [inline draft-irtf-nmrg-network-digital-twin-arch.md:274]:
```suggestion
effective changes might be implemented in the real network (i.e., subject to adequate validation and control checks). Digital twin
```

File: draft-irtf-nmrg-network-digital-twin-arch.md [hunk 6]
```diff
@@ -257,17 +257,26 @@ in the digital world.  It has the characteristics of virtual-reality
...
+for networks also play a crucial role in root cause analysis,
+providing a sandbox for testing hypotheses and validating the outcomes
```
Comment (boucadair) [inline draft-irtf-nmrg-network-digital-twin-arch.md:276]:
```suggestion
providing a sandbox for assessing hypotheses and validating the outcomes
```

File: draft-irtf-nmrg-network-digital-twin-arch.md [hunk 7]
```diff
@@ -257,17 +257,26 @@ in the digital world.  It has the characteristics of virtual-reality
...
+of data-driven insights without impacting end users. Therefore,digital
```
Comment (boucadair) [inline draft-irtf-nmrg-network-digital-twin-arch.md:277]:
```suggestion
of data-driven insights without impacting end users, when adequate isolation guards are in place. Therefore, digital
```

File: draft-irtf-nmrg-network-digital-twin-arch.md [hunk 8]
```diff
@@ -417,17 +426,17 @@ technologies such as network planning, construction, maintenance and
...
-diagnose, emulate, and control the real network based upon data,
-models, and interfaces. To that aim, a real-time and interactive
-mapping is required between the real network and its virtual twin
-network. Whether a Digital Twin supports all or a subset of the functions above (i.e., analyze,
-diagnose, emulate, and control) is deployment specific.
+## Characteristics of Network Digital Twin  {#def}
+
+So far, there is no standard definition for characteristic of "network
+digital twin" within the networking industry.  This document introduces
+four key elements (i.e.,data, models, mapping,and interfaces) to
```
Comment (boucadair) [inline draft-irtf-nmrg-network-digital-twin-arch.md:433]:
```suggestion
four key elements (i.e., data, models, mapping, and interfaces) to
```

File: draft-irtf-nmrg-network-digital-twin-arch.md [hunk 9]
```diff
@@ -417,17 +426,17 @@ technologies such as network planning, construction, maintenance and
...
+characterize the network digital twin. These four elements can be
+integrated with the network mangement system to analyze,diagnose, emulate,
```
Comment (boucadair) [inline draft-irtf-nmrg-network-digital-twin-arch.md:435]:
```suggestion
integrated into a network management system to analyze, diagnose, emulate,
```

**Review by boucadair — CHANGES_REQUESTED** [review]

**Review by cheneyzhoucheng — COMMENTED** [review]
Agree to the changes, as well as Med's update

**Pull request merged**